### PR TITLE
magazinerepack stringtable fix #5390

### LINKED ISSUE #5390: german language fail at magazinerepack
**Arma 3 Version:** `stable`
**CBA Version:** `stable`
**ACE3 Version:** `stable`

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
- https://github.com/acemod/ACE3/blob/master/addons/magazinerepack/stringtable.xml#L135
- should be: %1 volle Magazine und %2 angefangene Magazine
- ALSO, you can only repack one type of magazine so there is 0 or 1 (most time) partial mag anytime, so the number for partial mags is not needed

**Steps to reproduce:**
- repack magazines

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer) / Virtual Arsenal

**Placed Modules:**
- all

**RPT log file:**
- is not needed

### PR DESCRIPTION
**Pull request by shukari**
stringtable optimization

- fixes #5390

### CODE REVIEW

File: addons/magazinerepack/stringtable.xml [hunk 1]
```diff
@@ -132,7 +132,7 @@
         <Key ID="STR_ACE_MagazineRepack_RepackedMagazinesCount">
             <English>%1 Full and %2 Partial</English>
             <French>%1 plein(s) et %2 partiel(s)</French>
-            <German>%1 volle Magazine und 1 Magazin mit %2 Partronen</German>
+            <German>%1 volle Magazine und %2 angefangenes Magazine</German>
```
Comment (commy2) [inline addons/magazinerepack/stringtable.xml]: Maybe `Magazin(e)` so it works with "1"? Same for `volle(s)` und `angefangene(s)`.

Comment (shukari) [inline addons/magazinerepack/stringtable.xml]: In 90% of the cases you have >1 full and 1 partial... so it is optimised for the 90%... (e) looks so ugly or?

**Comment by shukari** [comment]
This issue is so small but i find no really good translation... here some optinions:

With ():
%1 volle(s) Magazin(e) und %2 angefangene(s) Magazin(e)
%1 volle(s) und %2 angefangene(s) Magazin(e)
%2 angefangene(s) und %1 volle(s) Magazin(e)

With the prediction for the most cases (full >1 and partial = 1):
%2 angefangenes und %1 volle Magazine -> favorite, because "%1 volle Magazine" is the important info
%1 volle und %2 angefangenes Magazine

**Comment by shukari** [comment]
I changed the translation to @commy2 suggestion and changed the message for successful repacking to full mags and x rounds left.
I hope that fits for all.

**Comment by jonpas** [comment]
@commy2 ?

File: addons/magazinerepack/stringtable.xml [hunk 2]
```diff
@@ -144,6 +96,6 @@
             <Korean>%1 꽉찼고 %2 부분참</Korean>
             <Chinesesimp>%1个满的与%2个部分的</Chinesesimp>
             <Chinese>%1個滿的與%2個部分的</Chinese>
-        </Key>
+          </Key>
```
Comment (commy2) [inline addons/magazinerepack/stringtable.xml]: spaces

Comment (shukari) [reply addons/magazinerepack/stringtable.xml]: fixed

**Pull request merged**